common/lib/dependabot/pull_request_creator/azure.rb

```diff
@@ -46,9 +47,7 @@ def azure_client_for_source
       end
 
       def branch_exists?
-        @branch_ref ||= azure_client_for_source.branch(branch_name)
-
-        @branch_ref
```

Comment:
Is this change intentional?

Reply:
Yes, RubyMine highlighted this as an unnecessary store before return. It's now the same implementation as the GitHub client, however the GitLab and Code Commit clients still use this pattern.

Reply:
ah yes it's only used once from `create` so seems good to me 👍